weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
```

Comment:
Doc string should explain completely what module does. Preferably That would not be a continuous sentence.

Reply:
Done.